app/jobs/post_grouper.rb

```diff
@@ -0,0 +1,19 @@
+#class PostGrouper < ActiveJob::Base
+class PostGrouper
+  def perform(cleanup_gene_claims = true)
```

Comment:
Yeah, I saw that, and didn't think that it mattered. Since Entrez doesn't even have gene claims, it's kind of moot, right?